packages/teams-js/src/public/webStorage.ts

```diff
@@ -25,12 +27,34 @@ export namespace webStorage {
       throw errorNotSupportedOnPlatform;
     }
 
+    if (
+      (GlobalVars.hostClientType === HostClientType.android || GlobalVars.hostClientType === HostClientType.ios) &&
+      (await getHostName()) === HostName.teams
+    ) {
+      // On Teams Mobile, they haven't yet implemented this capability. However, for compatibility reasons, we need
+      // to act as if they do. If they did implement it, they would return true, so that's what we do here.
+      // Getting Teams Mobile to implement this is a work-in-progress. Once they do implement it, we can remove this
+      // whole if-block. Until then, we cannot send the message to them because they will not understand it.
+      return true;
+    }
+
     return await sendAndUnwrap(
```

Comment:
I don't have a good idea here (yet), but it would be great to only return true for these mobile platforms *if* they aren't providing their own implementation. It mightbe possible to look at what happens when we receive a response from the host and return true if we're on a mobile platform AND we receive a specific error? Alternatively, we could build in a sort of "default" implementation of this capability on the host sdks for the mobile platform (although that would be more work).

I don't know that this is necessary, but it would be nice to avoid a bug in the future where mobile hosts implement this and return false, but we are still hardcoding the answer to true.

Reply:
Let me know how you feel about the latest iteration of this where I tried to achieve what you were looking for.